Add new_ones symbolic (#59255)

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#59539 Add new_ones symbolic (#59255)**
* #59538 Symbolic for ReLu6 (#58560)
* #59537 [ONNX] Sum empty tensor could not be exported to ONNX successfully. (#58141)
* #59536 [ONNX] Add support for torch.bernoulli() export (#57003)
* #59535 fix error message for type_as (#57948)
* #59534 Update Autograd Export Docs (#56594)

Add new_ones symbolic in PT-ONNX exporter

Co-authored-by: Shubham Bhokare <shubhambhokare@gmail.com>

Differential Revision: [D29046603](https://our.internmc.facebook.com/intern/diff/D29046603)

shubhambhokare1 committed:
Add new_ones symbolic (#59255)

Add new_ones symbolic in PT-ONNX exporter

Co-authored-by: Shubham Bhokare <shubhambhokare@gmail.com>

[ghstack-poisoned]

BowenBao committed:
Update on "Add new_ones symbolic (#59255)"


Add new_ones symbolic in PT-ONNX exporter

Co-authored-by: Shubham Bhokare <shubhambhokare@gmail.com>

[ghstack-poisoned]

SplitInfinity commented:
@splitinfinity has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29046603).

Commit:
Update on "Add new_ones symbolic (#59255)"


Add new_ones symbolic in PT-ONNX exporter

Co-authored-by: Shubham Bhokare <shubhambhokare@gmail.com>

Differential Revision: [D29046603](https://our.internmc.facebook.com/intern/diff/D29046603)

[ghstack-poisoned]
